src/Samples/SubModel/Program.fs

```diff
@@ -36,39 +71,24 @@ module Clock =
 module CounterWithClock =
 
   type Model =
-    { Count: int
-      StepSize: int
+    { Counter: Counter.Model
       Clock: Clock.Model }
 
   let init =
-    { Count = 0
-      StepSize = 1
+    { Counter = Counter.init ()
```

Comment:
Come to think of it, I'm thinking that since `init` is pure and does not depend on anything, perhaps it should simply be a value, not a function. What do you think? Perhaps this should be modified in all samples? (Could probably be a separate PR)

Reply:
Yes, I prefer making it a value.  In fact, this just like by question about `bindings` #107.  I also agree with making this change in a separate PR.

Do you prefer still calling it `init`?  In my project at work, I have been calling this `initModel` (which is short for "initial model" instead of "initialize (model)", but I am not completely sold on the name.

Reply:
Let's stick to `init`. 👍